en/matrix.md

```diff
@@ -0,0 +1,371 @@
+    <td style="border-style:hidden;"></td>
+  </tr>
+</table>
+
+<table id="feature-tracker">
+
+  <thead>   <!--  the <thead> tag is used to define the header section of a table -->
+  <tr>
+    <th rowspan="2">Product <br/>/ <br/>Service </th>
+    <th rowspan="2">Category</th>
+    <th rowspan="2">Key <br/> Features / <br/>Use Cases<br/><font size="1"><i class="italic">(hover)</i></font></th>
+    <th class="col-platform" style="display: none;color:blue;" colspan="3">Platform</th>
+    <th class="col-hwinterface" style="display: none;" rowspan="2"><a href="https://bitcoinops.org/en/topics/hwi/">HWI</a></th>
+
+    <th rowspan="2">Default <br/>Receive<br/> Address</th>
+
+    <th colspan="3" style="color:blue;">Native Segwit</th>
+    <th class="col-feebumping" style="display: none;color:blue;" colspan="4">Fee Bumping </th>
+
+    <th class="col-descriptors" style="display: none;" rowspan="2"><a href="https://bitcoinops.org/en/topics/output-script-descriptors/">Descriptors</a></th>
+
+    <th class="col-multipartytxn" style="display: none;color:blue;" colspan="6">Multi-Party Transactions </th>
+
+    <th class="col-pc_silentpayments" style="display: none;color:blue;" colspan="1"><div class="tooltip-container">
+    <a href="https://github.com/bitcoin/bips/blob/master/bip-0047.mediawiki">Payment <br>Codes</a><span class="tooltip">BIP47</span>
+    </div></th>
+
+    <th class="col-pc_silentpayments" style="display: none;color:blue;" colspan="2"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/silent-payments/">Silent <br>Payments</a><span class="tooltip">BIP352</span>
+    </div></th>
+
+    <th class="col-lightning" style="display: none;color:blue;" colspan="6"><div class="tooltip-container">Lightning<span class="tooltip">Lightning only sending (Pay Invoice) and receiving (Create Invoice) capabilities are considered.</span>
+    </div></th>
+
+    </tr>
+    <tr>
+
+    <th class="col-platform" style="vertical-align:top;display: none;"><div class="tooltip-container">Device<span class="tooltip">Desktop/Mobile</span>
+    </div></th>
+    <th class="col-platform" style="vertical-align:top;display: none;"><div class="tooltip-container">OS<span class="tooltip">Linux, Windows, OS, Android, iOS</span>
+    </div></th>
+    <th class="col-platform" style="vertical-align:top;display: none;"><div class="tooltip-container">Web /<br> Browser<span class="tooltip">Y/N</span>
+    </div></th>
+
+    <th style="vertical-align:top;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/bech32/">Bech32 <br/><font color="grey">P2WPKH<br/>P2WSH</font></a><span class="tooltip">BIP173</span>
+    </div></th>
+
+    <th style="vertical-align:top;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/bech32/">Bech32m <br/><font color="grey">P2TR</font></a><span class="tooltip">BIP350</span>
+    </div></th>
+
+    <th style="vertical-align:top;">Native <br/>Segwit<br/> Change</th>
```

Comment:
"Native Segwit Change" could use a tooltip

Reply:
Got it.

Reply:
To be more concrete, P2WPKH, P2WSH, and P2TR all belong to the category of native segwit outputs, but sometimes people use "native segwit" only to refer to P2WPKH and P2WSH while they call P2TR a "taproot output". Sometimes people even think that only P2WPKH is called "native segwit". Therefore, it would be helpful to clarify what the content of this column refers to specifically.